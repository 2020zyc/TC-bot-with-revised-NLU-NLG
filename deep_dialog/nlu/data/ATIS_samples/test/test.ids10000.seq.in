28 1 1 2 21 1 18 3 1 2 1 1 1 1 1 1 1 1 1
5 1 19 28 17 1 1 3 1 2 26 1 1 1 1 1
5 1 19 28 17 1 18 1 3 1 2 26 1
28 1 1 1 18 1 1 1 3 1 2 26 1 5 1 19
28 1 1 1 18 3 1 2 1 1 1 1 1 19 5 1 1
