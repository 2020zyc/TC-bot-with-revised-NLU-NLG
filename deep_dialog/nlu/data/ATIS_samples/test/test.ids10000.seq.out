2 2 2 2 2 2 2 2 4 2 3 10 2 2 2 2 2 1 1
2 7 13 2 2 2 2 2 4 2 3 10 2 1 1 1
2 7 13 2 2 2 2 2 2 4 2 3 10
2 2 2 2 2 2 5 6 2 4 2 3 10 2 7 13
2 2 2 2 2 2 4 2 3 10 10 2 7 13 2 1 1
